app/script/view_model/content/PreferencesDevicesViewModel.js

```diff
@@ -90,11 +70,8 @@ z.viewModel.content.PreferencesDevicesViewModel = class PreferencesDevicesViewMo
 
   updateDeviceInfo() {
     if (this.currentClient() && !this.localFingerprint().length) {
-      const {location, time} = this.currentClient();
+      const {time} = this.currentClient();
```

Comment:
You probably don't need a tmp variable for that. The two lines should fit into one